packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -51,6 +48,16 @@ const ResetPasswordWithRecoveryKeyVerified = ({
     navigate('/settings', { replace: true });
   };
 
+  const [isSync, setIsSync] = useState<boolean>();
+  useEffect(() => {
+    (async () => {
+      setIsSync(await relier.isSync());
+    })();
+  });
+
+  const serviceName =
+    relier.service === 'sync' ? MozServices.FirefoxSync : MozServices.Default;
```

Comment:
When we address the OAuth flow we'll want to change this but seems fine for now!

Though... one hiccup I see is that we aren't passing parameters to `resetPassword`. If I launch `fxa-dev-launcher` and click "Looking for Sync?" then click on "reset password" and add `showReactApp=true`, the parameters are lost except for the ones we specify in `router.js`, so I see "account settings" still.

We need to pass the needed parameters [here](https://github.com/mozilla/fxa/blob/d66211693dfb4773d859e74dceb139daf812ecee/packages/fxa-content-server/app/scripts/lib/router.js#L304) as [additionalParams](https://github.com/mozilla/fxa/blob/d66211693dfb4773d859e74dceb139daf812ecee/packages/fxa-content-server/app/scripts/lib/router.js#L495).

Reply:
@dschom I haven't checked your newest push just yet but this might be related that I see we need to do this in `ResetPassword` too, [see this line](https://github.com/mozilla/fxa/blob/e52ab767bd34a0c82666631b561ae82b527f74a4/packages/fxa-settings/src/pages/ResetPassword/index.tsx#L45)

Reply:
The ticket specifically called out this page view, so I didn't go beyond that. I started to apply the same change to the ResetPassword view, only to realize there's a bunch of places where this `serviceName` parameter is not wired up. A similar state exists for other relier parameters like `isSync`.

I think we should should land this PR first to establish the pattern to use, and then file an appropriately sized ticket for the remainder of the work.

By the way, the other way to go about these changes would be to call `CreateRelier()` in `App.tsx` and then prop drill. I believe we discussed having the pages pull in the context they depend on though, so I think the pattern here is what we should go with despite requiring more modification to story book and test mocks.